### PR TITLE
Rename process to workflow

### PR DESCRIPTION
**Pull request by antho1404**
Related to https://github.com/liteflow-labs/company/issues/54

Rename all notions of process to workflow

### CODE REVIEW

**Review by NicolasMahe — APPROVED** [review]

File: content/en/services/custom.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ This will create a `liteflow.yml` file in your current directory with the follow
 | ----------------- | --------------------------------- | ------------------------------------------------------------------------ |
 | **name**          | `String`                          | Name of the service to easily identify it.                               |
 | **description**   | `String`                          | (optional) Description of the service.                                   |
-| **tasks**         | `map<id,`[`Task`](#task)`>`       | Tasks that the service can process.                                      |
+| **tasks**         | `map<id,`[`Task`](#task)`>`       | Tasks that the service can workflow.                                     |
```
Comment (EmmanuelDrouin) [inline content/en/services/custom.md:64]: Workflow is not a verb it's a noun. So it's not working here.

File: content/en/services/custom.md [hunk 2]
```diff
@@ -229,9 +229,9 @@ repository: https://github.com/org/repo
 
 ## Create a task
 
-Services need to receive a request for an execution from a [process](/process) to execute any desired tasks. Every time a request for execution is received, the service is automatically called with the correct function.
+Services need to receive a request for an execution from a [workflow](/workflow) to execute any desired tasks. Every time a request for execution is received, the service is automatically called with the correct function.
 
-The service doesn't need to handle any communication to be able to process a task.
+The service doesn't need to handle any communication to be able to workflow a task.
```
Comment (EmmanuelDrouin) [inline content/en/services/custom.md:234]: Workflow is not a verb it's a noun. So it's not working here.

File: content/en/services/custom.md (hunk 1 above)
Comment (antho1404) [reply content/en/services/custom.md:64]: fixed and deployed in https://github.com/liteflow-labs/docs/pull/59

File: content/en/services/custom.md (hunk 2 above)
Comment (antho1404) [reply content/en/services/custom.md:234]: fixed and deployed in https://github.com/liteflow-labs/docs/pull/59

**Review by EmmanuelDrouin — COMMENTED** [review]
@antho1404 thanks for merging before i finalize my review.. Btw i added 2 comments about a wrong use of the word "workflow" as a verb

**Pull request merged**